src/main/java/org/folio/circulation/domain/policy/LoanPolicy.java

```diff
@@ -185,7 +186,7 @@ private ValidationError errorForNotMatchingOverrideCases() {
   private ValidationError errorForPolicy(String reason) {
     HashMap<String, String> parameters = new HashMap<>();
     parameters.put("loanPolicyId", getId());
-    parameters.put("loanPolicyName", getName());
+    parameters.put(LoanProperties.LOAN_POLICY_NAME, getName());
```

Comment:
Why has this constant been extracted to a property of the loan?

Reply:
In current implementation we have only two fields of a loan policy and they are both properties of a loan.

Reply:
Do you think we should remove it from loanProperties? 
Currently a loan response is as follows:
```
{
....
"loanPolicyId": "value",
"loanPolicyName": "value"
......
}
```
With regard to the representation, do you think that loan should look like the following:
loan:
```
{
  "loanPolicy": {
    "id": "value",
    "name": "value"
  }
}
```

Reply:
The loanpolicyName was added to loanproperties the same way as the loan policy id. Should this be changed now?

Reply:
The `loanPolicyId` is different to the name, it is deliberately stored as a reference to the last loan policy used in relation to that loan.

It cannot be moved without a breaking compatibility change.

I think the name can be included in an object. I was trying to decide whether to suggest that, it might make it clearer which properties are derived and read only (see the comment above), and would fit better the structure used in other places.

Something like:
`
"loanPolicyId": "b85eaf70-a9d2-429b-a1be-907254ced0ed"
"loanPolicy": {
  "name": "Example loan policy"
}
`